src/utils/copy-props.js

```diff
@@ -3,14 +3,14 @@ import { isObject } from './object'
 import identity from './identity'
 
 /**
- * Copies props from one array/object to a new aray/object. Prop values
- * are also cloned as new references to prevent possible mution of original
+ * Copies props from one array/object to a new array/object. Prop values
+ * are also cloned as new references to prevent possible mutation of original
  * prop object values. Optionally accepts a function to transform the prop name.
  *
  * @param {[]|{}} props
  * @param {Function} transformFn
  */
-export default function copyProps(props, transformFn = identity) {
+export default (props, transformFn = identity) => {
```

Comment:
Similar for util functions, it is handy for debugging to have function names appear in stack traces

Reply:
When ES6 is transpiled into ES5:

```js
const someVar = (a,b,c) => {
  // some stuff
}
// becomes (sorta...)
someVar = function /* anonymous */ (a, b, c) {
  // some stuff
}.bind(undefined) // or what ever the `this` context of the parent is
```

So any stack trace from an exception will never show a function name, where as providing a named function can be really handy in a debugging sense.

I'm not against ES6 function shorthands... but they do have their downfalls.

Reply:
Yeah. Code size is another thing we should take a look at.

Reply:
What way do we go about this now?

Reply:
For most of the one-liners (or ones with few lines), I think the ES6 shorthand is fine.

For ones that do something more complex (where there is a higher chance something could go wrong), I think maybe named functions might be useful.